build/media_source/plg_captcha_recaptcha/js/recaptcha.es6.js

```diff
@@ -8,10 +8,9 @@
   'use strict';
 
   window.JoomlainitReCaptcha2 = () => {
-    const elements = [].slice.call(document.getElementsByClassName('g-recaptcha'));
     const optionKeys = ['sitekey', 'theme', 'size', 'tabindex', 'callback', 'expired-callback', 'error-callback'];
 
-    elements.forEach((element) => {
+    document.getElementsByClassName('g-recaptcha').forEach((element) => {
```

Comment:
```suggestion
    document.querySelectorAll('.g-recaptcha').forEach((element) => {
```

Please don't use `getElementsByClassName`

Reply:
If you're sure there wasn't a reason for fetching a **live** HTMLCollection, then happy to move to static call.

Reply:
@C-Lodder if the liveliness of the method was used then this part would had to be on an event triggered from some DOM mutation. Reading the code that's not what's happening here